internal/infrastructure/kubernetes/bootstrap.yaml

```diff
@@ -0,0 +1,56 @@
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: 19000
+dynamic_resources:
+  cds_config:
+    resource_api_version: V3
+    api_config_source:
+      api_type: GRPC
+      transport_api_version: V3
+      grpc_services:
+      - envoy_grpc:
+          cluster_name: xds_cluster
+      set_node_on_first_message_only: true
+  lds_config:
+    resource_api_version: V3
+    api_config_source:
+      api_type: GRPC
+      transport_api_version: V3
+      grpc_services:
+      - envoy_grpc:
+          cluster_name: xds_cluster
+      set_node_on_first_message_only: true
+node:
+  cluster: envoy-gateway-system
+  id: envoy-default
+static_resources:
+  clusters:
+  - connect_timeout: 1s
+    load_assignment:
+      cluster_name: xds_cluster
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: {{ .XdsServerAddress }}
```

Comment:
the linter is complaining about this line
```
internal/infrastructure/kubernetes/bootstrap.yaml
  39:28     error    too many spaces inside braces  (braces)
  39:46     error    too many spaces inside braces  (braces)
```
but afaik those spaces are needed for templating to work

Reply:
@arkodg should we update [golangci.yml](https://github.com/envoyproxy/gateway/blob/main/tools/linter/golangci-lint/.golangci.yml) to skip this file, e.g.:
```
run:
  ...
  skip-files:
    - internal/infrastructure/kubernetes/bootstrap.yaml
```

Reply:
@danehans renamed the file to `*.tpl`, which I think it's better than add `skip-files` rule.

Reply:
lol, nice workaround, we cant validate the file anymore but it at least improves readability